app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1030,6 +1030,13 @@ def validate(self, data):
         return data
 
     def update(self, experiment, validated_data):
+        population_percent = validated_data.get("population_percent")
+        if population_percent != experiment.population_percent and experiment.status in (
+            NimbusConstants.Status.LIVE
+        ):
+            experiment.publish_status = NimbusConstants.PublishStatus.DIRTY
+            experiment.save()
+
         self.changelog_message = validated_data.pop("changelog_message")
```

Comment:
I'm not sure if `update` is the best place for this 🤔 I considered a few other places but didn't think that a validation method would make sense, and this made it clear that the other data was validated before we updated the `publish_status` (though would it make sense to make this change and _then_ validate that the `publish_status` is still valid once we make it `DIRTY`??)